lib/KaApi.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+// Copyright 2022. Plesk International GmbH. All rights reserved.
+
+namespace WHMCS\Module\Server\Plesk360Monitoring;
+
+use GuzzleHttp\Client;
+
+class KaApi
+{
+    private const CONTENT_TYPE_JSON = 'application/json';
+
+    private $client;
```

Comment:
Depends on the minimum PHP version a typed property could be used and maybe also `readonly`

PHP 7.4:
```suggestion
    private Client $client;
```
PHP 8.1:
```suggestion
    private readonly Client $client;
```

Reply:
Fixed